<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>745</width>
    <height>599</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>串口设置</string>
       </attribute>
       <layout class="QFormLayout" name="formLayout">
        <item row="0" column="0">
         <layout class="QVBoxLayout" name="verticalLayout_6">
          <item>
           <widget class="QWidget" name="widget_2" native="true">
            <layout class="QVBoxLayout" name="verticalLayout_2">
             <item>
              <widget class="QPushButton" name="openButton">
               <property name="text">
                <string>打开串口</string>
               </property>
              </widget>
             </item>
             <item>
              <layout class="QFormLayout" name="formLayout_6">
               <item row="0" column="0">
                <widget class="QLabel" name="label_3">
                 <property name="sizePolicy">
                  <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
                   <horstretch>0</horstretch>
                   <verstretch>0</verstretch>
                  </sizepolicy>
                 </property>
                 <property name="maximumSize">
                  <size>
                   <width>16777215</width>
                   <height>16777215</height>
                  </size>
                 </property>
                 <property name="text">
                  <string>端口号：</string>
                 </property>
                </widget>
               </item>
               <item row="0" column="1">
                <widget class="QComboBox" name="PortBox"/>
               </item>
              </layout>
             </item>
             <item>
              <layout class="QFormLayout" name="formLayout_2">
               <item row="0" column="0">
                <widget class="QLabel" name="label_4">
                 <property name="text">
                  <string>波特率：</string>
                 </property>
                </widget>
               </item>
               <item row="0" column="1">
                <widget class="QComboBox" name="BaudBox">
                 <item>
                  <property name="text">
                   <string>1200</string>
                  </property>
                 </item>
                 <item>
                  <property name="text">
                   <string>2400</string>
                  </property>
                 </item>
                 <item>
                  <property name="text">
                   <string>4800</string>
                  </property>
                 </item>
                 <item>
                  <property name="text">
                   <string>9600</string>
                  </property>
                 </item>
                 <item>
                  <property name="text">
                   <string>19200</string>
                  </property>
                 </item>
                 <item>
                  <property name="text">
                   <string>115200</string>
                  </property>
                 </item>
                 <item>
                  <property name="text">
                   <string>921600</string>
                  </property>
                 </item>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <layout class="QFormLayout" name="formLayout_3">
               <item row="0" column="0">
                <widget class="QLabel" name="label_5">
                 <property name="text">
                  <string>数据位：</string>
                 </property>
                </widget>
               </item>
               <item row="0" column="1">
                <widget class="QComboBox" name="BitNumBox">
                 <item>
                  <property name="text">
                   <string>8</string>
                  </property>
                 </item>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <layout class="QFormLayout" name="formLayout_4">
               <item row="0" column="0">
                <widget class="QLabel" name="label_6">
                 <property name="text">
                  <string>校验位：</string>
                 </property>
                </widget>
               </item>
               <item row="0" column="1">
                <widget class="QComboBox" name="ParityBox">
                 <item>
                  <property name="text">
                   <string>0</string>
                  </property>
                 </item>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <layout class="QFormLayout" name="formLayout_5">
               <item row="0" column="0">
                <widget class="QLabel" name="label_7">
                 <property name="text">
                  <string>停止位：</string>
                 </property>
                </widget>
               </item>
               <item row="0" column="1">
                <widget class="QComboBox" name="StopBox">
                 <item>
                  <property name="text">
                   <string>1</string>
                  </property>
                 </item>
                 <item>
                  <property name="text">
                   <string>2</string>
                  </property>
                 </item>
                </widget>
               </item>
              </layout>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QWidget" name="widget" native="true">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>200</width>
              <height>0</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>400</width>
              <height>300</height>
             </size>
            </property>
            <widget class="QPushButton" name="edit_pushButton">
             <property name="geometry">
              <rect>
               <x>50</x>
               <y>240</y>
               <width>75</width>
               <height>23</height>
              </rect>
             </property>
             <property name="text">
              <string>  确定修改</string>
             </property>
            </widget>
            <widget class="QLabel" name="label_8">
             <property name="geometry">
              <rect>
               <x>9</x>
               <y>9</y>
               <width>66</width>
               <height>16</height>
              </rect>
             </property>
             <property name="text">
              <string> 通道名称：</string>
             </property>
            </widget>
            <widget class="QLabel" name="label_9">
             <property name="geometry">
              <rect>
               <x>140</x>
               <y>10</y>
               <width>54</width>
               <height>12</height>
              </rect>
             </property>
             <property name="text">
              <string>倍率</string>
             </property>
            </widget>
            <widget class="QWidget" name="layoutWidget">
             <property name="geometry">
              <rect>
               <x>10</x>
               <y>28</y>
               <width>122</width>
               <height>206</height>
              </rect>
             </property>
             <layout class="QHBoxLayout" name="horizontalLayout_4">
              <item>
               <layout class="QVBoxLayout" name="verticalLayout_3">
                <item>
                 <widget class="QLabel" name="ch01_label">
                  <property name="text">
                   <string>CH 01</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLabel" name="ch02_label">
                  <property name="text">
                   <string>CH 02</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLabel" name="ch03_label">
                  <property name="text">
                   <string>CH 03</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLabel" name="ch04_label">
                  <property name="text">
                   <string>CH 04</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLabel" name="ch05_label">
                  <property name="text">
                   <string>CH 05</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLabel" name="ch06_label">
                  <property name="text">
                   <string>CH 06</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLabel" name="label_14">
                  <property name="text">
                   <string>CH 07</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLabel" name="label_15">
                  <property name="text">
                   <string>CH 08</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
              <item>
               <layout class="QVBoxLayout" name="verticalLayout_4">
                <item>
                 <widget class="QLineEdit" name="ch01_lineEdit">
                  <property name="maximumSize">
                   <size>
                    <width>80</width>
                    <height>16777215</height>
                   </size>
                  </property>
                  <property name="text">
                   <string>ch01</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLineEdit" name="ch02_lineEdit">
                  <property name="maximumSize">
                   <size>
                    <width>80</width>
                    <height>16777215</height>
                   </size>
                  </property>
                  <property name="text">
                   <string>ch02</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLineEdit" name="ch03_lineEdit">
                  <property name="maximumSize">
                   <size>
                    <width>80</width>
                    <height>16777215</height>
                   </size>
                  </property>
                  <property name="text">
                   <string>ch03</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLineEdit" name="ch04_lineEdit">
                  <property name="maximumSize">
                   <size>
                    <width>80</width>
                    <height>16777215</height>
                   </size>
                  </property>
                  <property name="text">
                   <string>ch04</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLineEdit" name="ch05_lineEdit">
                  <property name="maximumSize">
                   <size>
                    <width>80</width>
                    <height>16777215</height>
                   </size>
                  </property>
                  <property name="text">
                   <string>ch05</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLineEdit" name="ch06_lineEdit">
                  <property name="maximumSize">
                   <size>
                    <width>80</width>
                    <height>16777215</height>
                   </size>
                  </property>
                  <property name="text">
                   <string>ch06</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLineEdit" name="ch07_lineEdit">
                  <property name="maximumSize">
                   <size>
                    <width>80</width>
                    <height>16777215</height>
                   </size>
                  </property>
                  <property name="text">
                   <string>ch07</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLineEdit" name="ch08_lineEdit">
                  <property name="maximumSize">
                   <size>
                    <width>80</width>
                    <height>16777215</height>
                   </size>
                  </property>
                  <property name="text">
                   <string>ch08</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
             </layout>
            </widget>
            <widget class="QWidget" name="layoutWidget">
             <property name="geometry">
              <rect>
               <x>140</x>
               <y>30</y>
               <width>51</width>
               <height>204</height>
              </rect>
             </property>
             <layout class="QVBoxLayout" name="verticalLayout_5">
              <item>
               <widget class="QLineEdit" name="mu01_lineEdit">
                <property name="text">
                 <string>1</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLineEdit" name="mu02_lineEdit">
                <property name="text">
                 <string>1</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLineEdit" name="mu03_lineEdit">
                <property name="text">
                 <string>1</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLineEdit" name="mu04_lineEdit">
                <property name="text">
                 <string>1</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLineEdit" name="mu05_lineEdit">
                <property name="text">
                 <string>1</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLineEdit" name="mu06_lineEdit">
                <property name="text">
                 <string>1</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLineEdit" name="mu07_lineEdit">
                <property name="text">
                 <string>1</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLineEdit" name="mu08_lineEdit">
                <property name="text">
                 <string>1</string>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </widget>
          </item>
         </layout>
        </item>
        <item row="0" column="1">
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <widget class="QLabel" name="label">
            <property name="text">
             <string>接收窗口</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QTextEdit" name="textEdit">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>发送窗口</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QTextEdit" name="textEdit_2"/>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout">
            <item>
             <widget class="QPushButton" name="clearButton">
              <property name="text">
               <string>清空接收</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="sendButton">
              <property name="text">
               <string>发送数据</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="saveData_pushButton">
              <property name="text">
               <string>  保存数据</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
        <item row="1" column="1">
         <widget class="QLabel" name="label_file_address">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_2">
       <attribute name="title">
        <string>图表</string>
       </attribute>
       <layout class="QHBoxLayout" name="horizontalLayout_8">
        <item>
         <widget class="QCustomPlot" name="qCustomPlot" native="true"/>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header>qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
